svelte/package.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "uplot-svelte",
+	"version": "1.1.5",
+	"description": "Svelte wrapper for uPlot that allows you to work with charts declaratively inside your favorite framework",
+	"author": "Kiril Panayotov <contact@zakrok.dev> (https://zakrok.dev)",
+	"contributors": [
+		{
+			"name": "Sergey Kalinichev",
+			"email": "kalinichev.so.0@gmail.com"
+		}
+	],
+	"license": "MIT",
+	"homepage": "https://github.com/skalinichev/uplot-wrappers#readme",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/skalinichev/uplot-wrappers.git"
+	},
+	"bugs": {
+		"url": "https://github.com/skalinichev/uplot-wrappers/issues"
+	},
+	"keywords": [
+		"Charts",
+		"uPlot",
+		"Svelte",
+		"SvelteKit",
+		"Wrapper"
+	],
+	"scripts": {
```

Comment:
Svelte uses Vite and not webpack. I hope this won't be an issue. I will try to move the scripts but I assume it would require to also put the vite.config.js in the root.